core-libs/setup/ssr/optimized-engine/optimized-ssr-engine.ts

```diff
@@ -492,30 +493,13 @@ export class OptimizedSsrEngine {
    * @returns the context of the request and error if occurred during parsing traceparent header
    * @private
    */
-  private getRequestContext(
-    request: Request
-  ): [ExpressServerLoggerContext, Error | null] {
-    let error: Error | null = null;
-    // request.headers['traceparent'] =
-    //   '00-0af7651916cd43dd8448eb211c80319c-b7ad6b7169203331-0';
+  private getRequestContext(): ExpressServerLoggerContext {
     const requestContext: ExpressServerLoggerContext = {
       uuid: randomUUID(),
       timeReceived: new Date().toISOString(),
     };
 
-    try {
-      const traceContext = parseTraceparent(request.get('traceparent'));
-      if (traceContext) {
-        requestContext.traceContext = traceContext;
-      }
-    } catch (e) {
-      error =
-        e instanceof Error
-          ? e
-          : new Error('Unexpected error during parsing traceparent header');
-    }
-
-    return [requestContext, error];
+    return requestContext;
```

Comment:
Now this method doesn't return request's context, but only generates a random uuid and timestamp - which is request-agnositc. Unless we find a better name for this method, maybe we should move it's body to inside the `updateWithRequestContext()` method?